src/thin_client.rs

```diff
@@ -99,29 +98,6 @@ impl ThinClient {
         ))
     }
 
-    pub fn create_vote_account(
```

Comment:
I removed these helpers to as they're not really needed and ensnare thin_client with unnecessary details about voting